logging/unit_tests/test__gax.py

```diff
@@ -412,7 +412,7 @@ def test_write_entries_w_extra_properties(self):
             'severity': SEVERITY,
             'labels': LABELS,
             'insertId': IID,
-            'timestamp': NOW,
+            'timestamp': NOW.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
Please use `google.cloud._helpersRFC3339_MICROS`, rather than the literal format string.  Even better would be to call `_datetime_to_rfc3339`.